go/pkg/client/tree_whitebox_test.go

```diff
@@ -0,0 +1,52 @@
+package client
```

Comment:
maybe `client_test`?

Reply:
I cannot, because I'm trying to test an exported function (hence `whitebox` in the filename). Otherwise I would just place this whole thing inside `tree_test.go`.

I do see another way of doing this in https://medium.com/@robiplus/golang-trick-export-for-test-aa16cbd7b8cd. Do you know which is preferred?

Reply:
Ah, I see.